### PR TITLE
LRAC-14909 Resend

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LRAC-14909

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss [hunk 1]
```diff
@@ -15,4 +15,4 @@
 			right: calc(50% - #{$arrowOffset});
 		}
 	}
-}
\ No newline at end of file
+}
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss:18]: Can you remove changes on this file?

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/components/distribution-card/AddPropertyForm.jsx [hunk 2]
```diff
@@ -116,7 +117,10 @@ const AddPropertyForm = ({
 
 	return (
 		<Card.Body className='add-property-form-root'>
-			<div className='row d-flex flex-column flex-grow-1'>
+			<div
+				className='row d-flex flex-column flex-grow-1'
+				id={Containers.DistributionBreakdownCard}
+			>
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/components/distribution-card/AddPropertyForm.jsx:123]: You should add the id on `<Card>` instead here. Can you update?

**Review by interaminense — CHANGES_REQUESTED** [review]
Can you see my request changes?

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss:18]: @interaminense just remove `}` in line 18?

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/components/distribution-card/AddPropertyForm.jsx (hunk 2 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/components/distribution-card/AddPropertyForm.jsx:123]: There's a parent component called `Distribution Card` on `contacts/components/distribution-card/index.tsx`   which is already passing an id in line 185. Should I add it again on this child component?

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss (hunk 1 above)
Comment (interaminense) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss:18]: Can't you only discard changes?

**Comment by fmarcio** [comment]
ci:forward

**Comment by fmarcio** [comment]
@ViniLopes20 could you please check if the failure on ci:test:relevant is related to this PR? thanks

**Comment by ViniLopes20** [comment]
Hey @fmarcio, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

**Comment by fmarcio** [comment]
ci:forward:force

**Comment by fmarcio** [comment]
Resent to https://github.com/liferay-ac/liferay-portal/pull/824

**Pull request closed without merging**